cartography/intel/semgrep/findings.py

```diff
@@ -86,8 +103,7 @@ def transform_sca_vulns(raw_vulns: List[Dict[str, Any]]) -> Tuple[List[Dict[str,
     for vuln in raw_vulns:
         sca_vuln: Dict[str, Any] = {}
         # Mandatory fields
-        unique_id = f"{vuln['repositoryName']}|{vuln['advisory']['ruleId']}"
-        sca_vuln["id"] = unique_id
+        sca_vuln["id"] = vuln["groupKey"]
```

Comment:
non-blocker: This changes the schema for id. Could you instead introduce groupKey as another property?